src/components/stepper/index.jsx

```diff
@@ -1,25 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import ReactStepper from 'react-stepper-horizontal';
 import { mapToCssModules } from '../utils';
 import './styles.scss';
 
 const propTypes = {
-  children: PropTypes.node,
   className: PropTypes.string,
   cssModule: PropTypes.object,
-  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
 };
 
 const defaultProps = {
-  tag: 'ul',
+  activeColor: '#03caa5',
+  circleFontSize: 12,
+  circleTop: 0,
+  completeBarColor: '#03caa5',
+  completeColor: '#03caa5',
+  defaultBarColor: '#d9d9db',
+  defaultColor: '#d9d9db',
+  defaultTitleColor: '#7a7a7c',
+  size: 27,
+  titleFontSize: 12,
 };
 
 const Stepper = (props) => {
   const {
     className,
     cssModule,
-    tag: Tag,
+    activeColor,
+    circleFontSize,
+    circleTop,
+    completeBarColor,
+    completeColor,
+    defaultBarColor,
+    defaultColor,
+    defaultTitleColor,
+    size,
+    titleFontSize,
```

Comment:
do these need to be destructured and then added manually? cant they not be destructured and then just be included in ...attributes?

Reply:
I guess I did it in case someone wanted to override the defaults.  I went ahead and added them in the returned component.
<ReactStepper
        activeColor="#03caa5"
        circleFontSize="12"
        circleTop="0"
        completeBarColor="#03caa5"
        completeColor="#03caa5"
        defaultBarColor="#d9d9db"
        defaultColor="#d9d9db"
        defaultTitleColor="#7a7a7c"
        size="27"
        titleFontSize="12"
        {...attributes}/>

Reply:
oh i totally missed that. up to you if you want them to be submittable as props with defaults, or hardcoded